tests/test_bbox.py

```diff
@@ -59,3 +60,26 @@ def test_project_pt_to_pixel_location():
     pixel_loc = _project_pt_to_pixel_location(pt, proj, img_height, img_width)
     assert pixel_loc[0] == 320
     assert pixel_loc[1] == 240
+
+
+def test_project_pt_to_pixel_location_orthographic():
+    pt = [0, 0, 0]
+    proj = numpy.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
+    img_height = 480
+    img_width = 640
+
+    pixel_loc = _project_pt_to_pixel_location_orthographic(
+        pt, proj, img_height, img_width
+    )
+    assert pixel_loc[0] == 320
+    assert pixel_loc[1] == 240
+
+    # more interesting case
+    pt = [0.3, 0, 0]
+    proj = numpy.array([[0.08951352, 0, 0], [0, 0.2, 0], [0, 0, -0.0020006]])
+
+    pixel_loc = _project_pt_to_pixel_location_orthographic(
+        pt, proj, img_height, img_width
+    )
+    assert pixel_loc[0] == 328
```

Comment:
How do you calculate the number 328? If we change the `img_height` and `img_width`, shall we also change the assertion?
You can add some simple calculations in the test code to automatically check it.

Reply:
Something like this?:
`assert pixel_loc[0] == int((proj[0][0] * pt[0] + 1) * 0.5 * img_width) # 328`
`assert pixel_loc[1] == img_width // 2`

Reply:
Exactly. If the assertion can be calculated easily, I recommend doing so.